### PR TITLE
Add new plugin for Appwrite

### PR DESCRIPTION
**Pull request by codercatdev**
Should we publish the npm repo?

## Description

This allows for an Appwrite source.

### CODE REVIEW

**Review by moonmeister — CHANGES_REQUESTED** [review]
I'm going to assume your actual app code is fine. I'm just looking at monorepo integration. You can delete the change log as noted there as well as the `package-lock.json` from npm since we use yarn. Yes, we can get this published to NPM, if you add a changeset, that should just happen. If not, I can handle getting it published manually. Thanks for building on the Gatsby UC repo!!!

File: packages/gatsby-source-appwrite/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# gatsby-source-supabase
```
[the remark this answers is not in the document]
Comment (codercatdev) [reply packages/gatsby-source-appwrite/CHANGELOG.md:1]: Getting stuck with yarn, is there a way to build this outside of my Mac?

image

[the remark this answers is not in the document]
Comment (moonmeister) [reply packages/gatsby-source-appwrite/CHANGELOG.md:1]: Those are all warnings. I'm not seeing any errors.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://github.com/mattcompiles"><img src="https://avatars.githubusercontent.com/u/8802980?v=4?s=100" width="100px;" alt=""/><br /><sub><b>mattcompiles</b></sub></a><br /><a href="https://github.com/gatsby-uc/plugins/commits?author=mattcompiles" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/klyngen"><img src="https://avatars.githubusercontent.com/u/14232560?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Martin Klingenberg</b></sub></a><br /><a href="https://github.com/gatsby-uc/plugins/commits?author=klyngen" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/codercatdev"><img src="https://avatars.githubusercontent.com/u/3102249?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Alex Patterson</b></sub></a><br /><a href="https://github.com/gatsby-uc/plugins/commits?author=codercatdev" title="Code">💻</a></td>
```
[the remark this answers is not in the document]
Comment (codercatdev) [reply README.md]: added

File: packages/gatsby-source-appwrite/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (codercatdev) [reply packages/gatsby-source-appwrite/CHANGELOG.md]: I believe I have this corrected now.

File: README.md [hunk 3]
```diff
@@ -57,6 +57,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://github.com/mattcompiles"><img src="https://avatars.githubusercontent.com/u/8802980?v=4?s=100" width="100px;" alt=""/><br /><sub><b>mattcompiles</b></sub></a><br /><a href="https://github.com/gatsby-uc/plugins/commits?author=mattcompiles" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/klyngen"><img src="https://avatars.githubusercontent.com/u/14232560?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Martin Klingenberg</b></sub></a><br /><a href="https://github.com/gatsby-uc/plugins/commits?author=klyngen" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/codercatdev"><img src="https://avatars.githubusercontent.com/u/3102249?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Alex Patterson</b></sub></a><br /><a href="#plugin-moonmeister" title="Plugin/utility libraries">🔌</a><a href="https://github.com/gatsby-uc/plugins/commits?author=codercatdev" title="Code">💻</a></td>
```
Comment (moonmeister) [inline README.md:60]: Sorry, I just realized you were adding this manually. This should be added using the `yarn ac` command from the root directory. It's the all-contirubutors CLI.

File: packages/gatsby-source-appwrite/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    <td valign="top"> <img height="100px" width="100px" src="https://res.cloudinary.com/practicaldev/image/fetch/s--hsQ1oxcF--/c_fill,f_auto,fl_progressive,h_320,q_auto,w_320/https://dev-to-uploads.s3.amazonaws.com/uploads/organization/profile_image/2225/18b2c66e-fba2-4f4f-8196-8184c599b75d.png"></td>
+  </tr>
+</table>
+
+Gatsby source plugin for building websites using Appwrite as a data source
+
+## How to install
+
+```
+npm install gatsby-source-appwrite appwrite
```
Comment (moonmeister) [inline packages/gatsby-source-appwrite/README.md:15]: Here you say to install `appwrite` but the peer dependency listed in the `package.json` is `node-appwrite`. Should these match?

**Review by moonmeister — CHANGES_REQUESTED** [review]
Almost there

File: README.md (hunk 3 above)
Comment (codercatdev) [reply README.md]: okay updated using the yarn ac command

File: packages/gatsby-source-appwrite/README.md (hunk 4 above)
Comment (codercatdev) [reply packages/gatsby-source-appwrite/README.md]: You are correct, I was hoping to use the isomorphic package, but cannot yet and had to fall back to node.

**Review by moonmeister — APPROVED** [review]

**Pull request merged**